app/controllers/service_uploads_controller.rb

```diff
@@ -45,15 +45,9 @@ def create
 
         return unless date_ended
 
-        discontinued_service = DiscontinuedService.new(
-          service: service,
-          discontinued_at: date_ended,
-          recorded_by_educator: current_educator,
-        )
+        service.update_attributes(:discontinued_at => date_ended, :discontinued_by_educator_id => current_educator.id)
 
-        if discontinued_service.valid?
-          discontinued_service.save
-        else
+        unless service.update_attributes(:discontinued_at => date_ended, :discontinued_by_educator_id => current_educator.id)
```

Comment:
Why does this repeat the code on line 48? We should only need to `update_attributes` once.

Reply:
So could we safely remove line 48? :D

Reply:
Happy to do some pairing on this together @Jngai!